addons/html_editor/static/src/main/table/table_plugin.js

```diff
@@ -460,6 +479,56 @@ export class TablePlugin extends Plugin {
         }
     }
 
+    onMousedown(ev) {
+        this._currentMouseState = ev.type;
+        this._lastMouseClickPosition = [ev.x, ev.y];
+        this.deselectTable();
+    }
+
+    onMouseup(ev) {
+        this._currentMouseState = ev.type;
+    }
+
+    onMousemove(ev) {
+        if (this._currentMouseState !== "mousedown" || this.shared.isResizingTable()) {
```

Comment:
@FrancoisGe This is kind of an intricate case here. Both the table and table_resize plugins are binding stuff on the mousemove and we don't want both code to activate at the same time. Here @adsa-odoo has solved the problem by depending on table_resize, but I think it's a little weird that table depends on table_resize, don't you think?

A solution I could see is to set a variable on mouse down if the mousedown happened in the cell and not on the borders. That would eliminate the dependency, but it would be kind of a hidden coupling between table and table_resize because one would check something like "did mousedown happen in _less_ than 5px radius around the border" while the other would check something like "did mousedown happen in _more_ than 5px radius around the border". If the value of 5 is not shared between the two and we ever change one but not the other, it's going to behave in unexpected ways. Obviously we could have the sensitivity radius be a shared constant variable in one of the two plugins, but conceptually it's kind of weird for it to be in table as it only exists _because_ the resize feature exists, otherwise there would be no need for a special sensititvity zone around the td borders, and if we put it into table_resize then we're back to table depending on table_resize. Any better idea or opinion? ^^'

Reply:
Or we can check whether `editable` contains `o_row_resize` OR `o_col_resize` class during mousemove ?

Reply:
Following our discussion with @FrancoisGe and @Goaman we decided to implement my suggestion which is to have an exported const in `table_plugin` that exports the value `5` so that both `table` and `table_resize` can use it. In `table_resize`, the mousedown has to happen within 5px around the border, this is already the case. In `table`, you will change the code so that the mousedown has to happen more than 5px away from the border for the selection code to be triggered. This way, only one of the two behaviors will be triggered at the same time.

Note that this introduces some sort of a hidden dependency between the two because of the coupling but we will handle that in another task to refactor the resize mechanism.